lib/animina/accounts/resources/user.ex

```diff
@@ -104,6 +106,12 @@ defmodule Animina.Accounts.User do
 
   actions do
     defaults [:create, :read, :update]
+
+    read :random_users do
```

Comment:
Why do we need this action?

Reply:
I've moved this to the PotentialPartner module. Its now part of the query and not an action